bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -77,11 +79,11 @@ spec:
           fi
 
           # Take over network functions on the node
-          rm -f /etc/cni/net.d/80-openshift-network.conf
+          rm -f /etc/cni/net.d/80-openshift-network.conf {{.CNIPath}}
```

Comment:
Maybe add a comment to keep the `rm -f 80-openshift-network.conf` for upgrade reasons?

Reply:
done